### PR TITLE
Show activity

### PR DESCRIPTION
**Pull request by hellovietnam93**

### CODE REVIEW

File: app/models/activity.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Activity < ActiveRecord::Base
   validates :target_id, presence: true
   validates :name, presence: true
 
+  scope :action, ->user{where "user_id = ?", user.id}
   private
```
Comment (ThuBM) [inline app/models/activity.rb]: add new line before this line

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -193,3 +193,40 @@ footer ul li {
...
+}
+
+.red
+{
+  left: 150px;
+  background-color: #E74C3C;
+  border-bottom: 5px solid #BD3E31;
+  text-shadow: 0px -2px #BD3E31;
+}
+.action-button:active
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: new line

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -1,10 +1,48 @@
...
+    <div class="col-md-9">
+      <div class="col-md-6 col-md-offset-2">
+        <%= link_to t("word.wordall"), words_path, class: "action-button shadow animate blue" %>
+        <%= link_to t("categories.lessons"), root_path, class: "action-button shadow animate red" %>
+      </div>
+      <div class="row">
+        <div class="col-md-12">
+          <div id="activities">
+            <ul class="list-group">
+              <li class="list-group-item active"><%= t("Act") %></li>
```
Comment (ThuBM) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/views/static_pages/home.html.erb [hunk 4]
```diff
@@ -1,10 +1,48 @@
...
+            </ul>
+          </div>
+          <%= will_paginate @activities %>
+        </div>
+      </div>
+    </div>
+  </div>
+<% else %>
+  <div class="center jumbotron">
+    <h1><%= t("FELS") %></h1>
```
Comment (ThuBM) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -1,10 +1,48 @@
...
+    <p>
+      <%= t("intro") %>
```
Comment (ThuBM) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -1,10 +1,48 @@
...
+      <a href="http://www.railstutorial.org/"><%= t("FELS") %></a>
+      <%= t("app") %>.
```
Comment (ThuBM) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/views/activities/_activity.html.erb [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<li class="list-group-item list-group-item-info">
+  <% if activity.name == Settings.activities.learned %>
+    <%= "hello" %>
```
Comment (trinhducduy) [inline app/views/activities/_activity.html.erb]: hello?

Comment (ThuBM) [inline app/views/activities/_activity.html.erb]: I18n

File: app/controllers/static_pages_controller.rb [hunk 8]
```diff
@@ -1,5 +1,14 @@
 class StaticPagesController < ApplicationController
   def home
+    if logged_in?
+      @words = Word.learned(current_user)
```
Comment (trinhducduy) [inline app/controllers/static_pages_controller.rb]: remove ()

**Comment by ThuBM** [comment]
OK

**Pull request merged**